extension/jaegerremotesampling/internal/remote_strategy_store.go

```diff
@@ -10,7 +10,6 @@ import (
 	"time"
 
 	grpcstore "github.com/jaegertracing/jaeger/cmd/agent/app/configmanager/grpc"
```

Comment:
this is still a dependency?

Reply:
I mistakenly thought this was out of scope. Fixed now